frontend/public/components/routes.tsx

```diff
@@ -267,6 +222,22 @@ const RouteTargetRow = ({route, target}) => <tr>
   <td>{calcTrafficPercentage(target.weight, route)}</td>
 </tr>;
 
+const RouteIngressStatus: React.SFC<RouteIngressStatusProps> = ({ingresses}) =>
+  <React.Fragment>
+    {_.map(ingresses, (ingress) =>
+      <React.Fragment key={ingress.host}>
```

Comment:
Sorry, my previous suggestion was wrong. This should be `key={ingress.routerName}`. `host` is not unique

Reply:
ok, fixed